homeduino.ino

```diff
@@ -1,8 +1,9 @@
 #include <SerialCommand.h>
 #include <OneWire.h>
 #include <DallasTemperature.h>
+#include <stdlib.h>
 
-#define ONE_WIRE_BUS 2
+#define ONE_WIRE_BUS 12
```

Comment:
That's correct
On zo 22 feb. 2015 at 09:07 sweet pi - sweet home <notifications@github.com>
wrote:

> In homeduino.ino
> <https://github.com/pimatic/homeduino/pull/11#discussion_r25130160>:
>
> >
> > -#define ONE_WIRE_BUS 2
> > +#define ONE_WIRE_BUS 12
>
> This defined isn't used anymore, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pimatic/homeduino/pull/11/files#r25130160>.
>